rre-maven-plugin/rre-maven-external-elasticsearch-plugin/src/main/java/io/sease/rre/maven/plugin/external/elasticsearch/RREvaluateMojo.java

```diff
@@ -87,7 +90,13 @@ public void execute() throws MojoExecutionException {
                     null,
                     persistence);
 
-            final Map<String, Object> configuration = Collections.emptyMap();
+            final Map<String, Object> configuration = new HashMap<>();
+            // Most of these don't actually apply except for plugins, but the class ExternalElastic extends expects them
+            configuration.put("path.home", "/tmp");
```

Comment:
Could the value for path.home be changed to `System.getProperty("java.io.tmpdir")`, for consistency across OSes. I appreciate it probably doesn't cause a directory to be created, but if we *do* decide to start the ES node in future, it may do so.